src/map/packets.h

```diff
@@ -2220,6 +2220,11 @@ packet(0x96e,-1,clif->ackmergeitems);
 	packet(0x0368,6,clif->pSolveCharName,2);
 	packet(0x0363,8,clif->pDull); // CZ_JOIN_BATTLE_FIELD
 	packet(0x0436,4,clif->pDull); // CZ_GANGSI_RANK
+	packet(0x0988, 6); //ZC_NOTIFY_CLAN_CONNECTINFO
+	packet(0x0989, 2); //ZC_ACK_CLAN_LEAVE
+	packet(0x098a, -1); //ZC_CLANINFO
```

Comment:
0x098a added in client 20120702, this mean move it to correct version

Reply:
@4144 Only the `0x098`, are the others in the right version?

Reply:
Yes only 0x098a.
```
#if PACKETVER >= 20120702
// new packets
	packet(0x098a,-1); // ZC_CLANINFO
#endif
```

Also add 0x0988, 0x0989 into separate block after shuffle packets something like this:
```
#if PACKETVER >= 20120716
// changed packet sizes
	packet(0x0988,6); // ZC_NOTIFY_CLAN_CONNECTINFO
	packet(0x0989,2); // ZC_ACK_CLAN_LEAVE
#endif
```

Reply:
Smoke said to serialize them in a packed struct like others map packets, so I think I won't need to do changes in `packets.h`.

Reply:
All packets must be in packets.h

Reply:
Oh okay then. Thanks :D

Reply:
`packets.h`